oak-doc/src/site/markdown/participating.md

```diff
@@ -15,29 +15,62 @@
-or patches. To subscribe to issue notifications, send a message to [oak-issues@]
-(mailto:oak-issues-subscribe@jackrabbit.apache.org). 
-
-The latest Oak sources are available on [GitHub][1]. To subscribe to commit notifications, 
-send a message to [oak-commits@] (mailto:oak-commits-subscribe@jackrabbit.apache.org).
-
-For more details related to various mailing list have a look at http://jackrabbit.apache.org/mailing-lists.html
-
-We generally follow a [CTR](https://www.apache.org/foundation/glossary.html#CommitThenReview) policy. 
-However it is up to each individual committer to pro-actively ask for a review of a patch on 
-oak-dev@ or to even call for a [RTC](https://www.apache.org/foundation/glossary.html#ReviewThenCommit). 
-Special care should be taken with backports to maintenance branches. Back ports bear a certain risk 
-of introducing regressions to otherwise stable branches. Each back ported change should be carefully 
-evaluated for its potential impact, risk and possible mitigation. It is the responsibility of each 
-committer to asses these and ask for advise or reviewing on oak-dev@ if uncertain. Whether using 
-RTC or CTR is up to the committer.
+or patches. To subscribe to issue notifications, send a message to
+[oak-issues@](mailto:oak-issues-subscribe@jackrabbit.apache.org).
+
+- On "trunk": when done with a ticket, set it to "resolved" and set "Fix Version"
+  to the next unreleased version. Our workflow does not allow re-opening a
+  closed ticket, so we close tickets only when a public release has been made
+  with that change. Until that happens, a ticket can always be re-opened
+  and further work can happen.
+- On maintenance branch (currently 1.22): re-use the existing Jira ticket an
+  just add to "Fix Version" (unless the backport is complex).
+- Add "Affects Version" and "Fix Version" as and when applicable (but not
+  otherwise).
+- Be careful not to include sensitive information (be it in descriptions,
+  attachments, or log files).
  
-[1]: https://github.com/apache/jackrabbit-oak
+## Source Code
+
+The latest Oak sources are available on [GitHub](https://github.com/apache/jackrabbit-oak).
+To subscribe to commit notifications, send a message to [oak-commits@](mailto:oak-commits-subscribe@jackrabbit.apache.org).
+
+### Making Changes
+
+#### Pull Requests (PRs)
+
+- Minimize PRs; do not modify whitespace/coding style except where needed. This
+  makes them much easier to review, also minimizes confusion when using
+  "git blame".
+- Structure tickets/PRs so that things that can be separated are (that can be
+  useful for backports and reverts).
+- Have test cases (when there's no immediate fix, create a ticket and a PR just
+  for the test and mark it "ignored", pointing to the actual issue).
+- PRs that contain multiple commits in general should be "squashed and merged".
+- When new files are added, make sure they have the proper license on it
+  (in doubt, run the build with "-Prat").
+
```

Comment:
+ a note about deleting branches after a PR was merged (unless it is worth keeping)?

Reply:
* plus I was also wondering about adding a comment about risks of force-push breaking github pull request conversations - not sure if force-push is the only reason why sometimes conversations get untangled from in the `/files` view, so not sure. maybe we could just add something like "force-push has the risk of breaking PR conversation flow"..?

Reply:
please suggest wording :-)

Reply:
```suggestion
- after merging a PR consider deleting the branch unless it should be kept
```